### PR TITLE
LIVE: UPDATE WIKI JAMULUS 3.6.1

### PR DESCRIPTION
**Pull request by ann0see**
@jamulussoftware/translators @jamulussoftware/wikiadmin 

This pull request will update jamulus.io with the latest changes.

### CODE REVIEW

**Comment by ann0see** [comment]
Not yet to be merged (since some translations might not be up to date yet).

**Comment by ann0see** [comment]
I propose to merge this on 2020-11-28 or 2020-11-29

**Comment by ann0see** [comment]
@gilgongo Merge tomorrow?

**Comment by gilgongo** [comment]
We can wait until the French are done and then do the merge. But it is my birthday tomorrow...

**Comment by gegeweb** [comment]
> We can wait until the French are done and then do the merge. But it is my birthday tomorrow...

It will take some time to review all the pages. Don't know how @trebmuh see the things. But the missing pages could be merged and published after the first ones yet merged, no? 

Happy brithday in advance!

**Comment by trebmuh** [comment]
I think waiting for all the French pages to be translated isn't a good idea.
We should merge what is already ready and the other French translated pages will go on a next round.

**Comment by ann0see** [comment]
Ok. So should we start a review of this PR (find typos etc.) and merge it tomorrow evening?

**Comment by trebmuh** [comment]
> So should we start a review of this PR (find typos etc.) and merge it tomorrow evening?

All good from my point ov view.
I should be able to do one or two more review for the French translation until tomorrow evening. Would that be an issue from your perspective?

File: wiki/de/de-Choosing-a-Server-Type.md [hunk 1]
```diff
@@ -45,14 +45,14 @@ Jamulus --nogui --server
...
 
-Ein normaler Server kann sich an deinem zentralen Server registrieren, um in deiner Liste angezeigt zu werden. Dazu muss dieser Server mit dem Flag `--centralserver` und deiner IP/Domain gestartet werden.
+Jeder Server kann sich an deinem zentralen Server registrieren, um in deiner Liste angezeigt zu werden. Dazu muss dann der anzumeldende Server mit dem Flag `--centralserver` und deiner IP/Domain gestartet werden.
 
 Der Server, der als zentraler Server konfiguriert werden soll, muss mit `--centralserver localhost` gestartet werden (d.h. der zentrale Server, an dem sich dieser Server anmeldet ist er selbst).
 
-Wenn du steuern möchtest, welche Server sich bei deinem zentralen Server registrieren können, kannst du eine Whitelist mit der Option `--listfilter` aktivieren. Weitere Informationen zu dieser Funktion findest du auf der Seite [Kommandozeilen-Optionen](Command-Line-Options).
```
Comment (ann0see) [inline wiki/de/de-Choosing-a-Server-Type.md:56]: typo

**Review by ann0see — COMMENTED** [review]
typo

**Review by ann0see — COMMENTED** [review]
Fixed typo

**Comment by ann0see** [comment]
> I should be able to do one or two more review for the French translation until tomorrow evening. Would that be an issue from your perspective?

No problem.

**Comment by trebmuh** [comment]
> > I should be able to do one or two more review for the French translation until tomorrow evening. Would that be an issue from your perspective?
> 
> No problem.

Perfect.
Note that I'm fairly confident in the French translations contained in this PR.

**Comment by gegeweb** [comment]
@trebmuh 
> I should be able to do one or two more review for the French translation until tomorrow evening.

I just reviewed myself:
* Installation for Windows
* Installation fo Linux

Redrafted again to fix some missed corrections to keep close as strictly possible to EN in `translation`.
Should not take a lot of time (I hope!) you can start by these.

**Comment by trebmuh** [comment]
@gegeweb please put that on the #38 thread where it belongs better than here.

**Comment by trebmuh** [comment]
@ann0see if it isn't to late yet, please merge in #70 before merging this very PR. Thanks.

**Comment by ann0see** [comment]
@jamulussoftware/translators I will merge this PR this evening. You still have about 1 to 2 hours for a final check.

**Comment by ann0see** [comment]
This PR is now merged!

**Comment by trebmuh** [comment]
:+1:

**Pull request merged**